internal/versions/versions.go

```diff
@@ -136,14 +366,14 @@ func (h *handler) GetReleaseImage(ctx context.Context, openshiftVersion, cpuArch
 				wg.Done()
 				h.sem.Release(1)
 			}()
-			existsInCache := false
+			existsInConfiguration := false
```

Comment:
I think don't change these names here because the config is acting as a cache in this case.